src/main/java/sonia/scm/jenkins/JenkinsConfiguration.java

```diff
@@ -54,8 +54,7 @@
   private String url;
   private String username;
   private boolean csrf;
-
-  //~--- get methods ----------------------------------------------------------
+  private Set<BuildParameter> buildParameters;
```

Comment:
A set is used here, but the referenced class does not have isEqual or hash code methods.